packages/tatum-bch/src/wallet/wallet.ts

```diff
@@ -21,11 +21,11 @@ export const generateBchWallet = async (testnet: boolean, mnem: string): Promise
 
 /**
  * Generate wallet
- * @param testnet testnet or mainnet version of address
  * @param mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param options.testnet optional testnet or mainnet version of address
  * @returns wallet or a combination of address and private key
  */
-export const generateWallet = (testnet: boolean, mnemonic?: string) => {
-  const mnem = mnemonic ? mnemonic : generateMnemonic(256)
-  return generateBchWallet(testnet, mnem)
+export const generateWallet = (mnemonic: string, options?: { testnet?: boolean }) => {
```

Comment:
mnemonic shoud be optional `...(mnemonic?: string)..`

Reply:
How come? I can see it here required.
https://github.com/tatumio/tatum-js/blob/d1de9b125a1b03ed72a6b3676dcadde9bb64891b/src/wallet/wallet.ts#L199